bin/development

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env ruby
 
-require File.join(File.dirname(__FILE__), '..', 'lib', 'parity')
+$LOAD_PATH.unshift File.expand_path(File.join("..", "..", "lib"), __FILE__)
+require "parity"
```

Comment:
I am going to assume not, but I will test regardless. I'm basing this off of some of the maintained ruby packages that currently exist in debian.